### PR TITLE
CAMEL-18694 - Fix documentation to use new path attribute instead of uri

### PR DESCRIPTION
**Pull request by apupier**
for rest configuration

### CODE REVIEW

**Review by oscerd — APPROVED** [review]

File: docs/user-manual/modules/ROOT/pages/rest-dsl.adoc [hunk 1]
```diff
@@ -214,13 +213,12 @@ And in XML:
...
-  <get uri="/customers/{id}">
+  <get path="/customers/{id}">
     <to uri="direct:customerDetail"/>
   </get>
-  <get uri="/customers/{id}/orders">
+  <get path="/customers/{id}/orders">
     <to uri="direct:customerOrders"/>
-  </get>
-  <post uri="/customers/neworder">
+  </gepost path=t uri="/customers/neworder">
```
Comment (essobedo) [inline docs/user-manual/modules/ROOT/pages/rest-dsl.adoc:221]: Seems to be a typo here

File: docs/user-manual/modules/ROOT/pages/rest-dsl.adoc [hunk 2]
```diff
@@ -141,13 +141,12 @@ only. The example above can be defined as:
...
-  <get uri="/customers/{id}">
+  <get path="/customers/{id}">
     <to uri="direct:customerDetail"/>
   </get>
-  <get uri="/customers/{id}/orders">
+  <get path="/customers/{id}/orders">
     <to uri="direct:customerOrders"/>
-  </get>
-  <post uri="/customers/neworder">
+  </get>post path=uri="/customers/neworder">
```
Comment (essobedo) [inline docs/user-manual/modules/ROOT/pages/rest-dsl.adoc:149]: Seems to be a typo here

File: docs/user-manual/modules/ROOT/pages/rest-dsl.adoc [hunk 3]
```diff
@@ -117,13 +117,13 @@ And using XML DSL it becomes:
...
-  <get uri="/{id}">
+  <get path="/{id}">
     <to uri="direct:customerDetail"/>
   </get>
-  <get uri="/{id}/orders">
+  <get path="/{id}/orders">
     <to uri="direct:customerOrders"/>
   </get>
-  <post uri="/neworder">
+ post path=i="/neworder">
```
Comment (essobedo) [inline docs/user-manual/modules/ROOT/pages/rest-dsl.adoc:126]: Seems to be a typo here

**Review by essobedo — APPROVED** [review]
Thx for the contribution, it looks like there are some typos to fix

File: docs/user-manual/modules/ROOT/pages/rest-dsl.adoc [hunk 4]
```diff
@@ -117,13 +117,13 @@ And using XML DSL it becomes:
...
-  <get uri="/{id}">
+  <get path="/{id}">
     <to uri="direct:customerDetail"/>
   </get>
-  <get uri="/{id}/orders">
+  <get path="/{id}/orders">
     <to uri="direct:customerOrders"/>
   </get>
-  <post uri="/neworder">
+  <post path=i="/neworder">
```
Comment (essobedo) [inline docs/user-manual/modules/ROOT/pages/rest-dsl.adoc:126]:
```suggestion
  <post path="/neworder">
```

**Comment by apupier** [comment]
> Thx for the contribution, it looks like there are some typos to fix

effectively, my search and replace for the `put` went completely bad. Sorry.

**Comment by essobedo** [comment]
There are remaining code snippet to fix at least in this article https://github.com/apache/camel/blob/main/docs/user-manual/modules/faq/pages/how-do-i-import-rests-from-other-xml-files.adoc

**Comment by apupier** [comment]
> There are remaining code snippets to fix at least in this article https://github.com/apache/camel/blob/main/docs/user-manual/modules/faq/pages/how-do-i-import-rests-from-other-xml-files.adoc

ah, I missed this other kind of change. I will search and replace for `rest uri=`

**Review by essobedo — APPROVED** [review]
LGTM 👍 thx

**Review by davsclaus — APPROVED** [review]

**Pull request merged**